test/Tests.Infrastructure/RavenTestBase.Indexes.cs

```diff
@@ -29,6 +30,14 @@ public IndexesTestBase(RavenTestBase parent)
         {
             _parent = parent ?? throw new ArgumentNullException(nameof(parent));
         }
+		
+		public async Task<PutIndexResult> CreateIndex(IDocumentStore store, AbstractIndexCreationTask index)
+        {
+            var results = await store.Maintenance.ForDatabase(store.Database).SendAsync(new PutIndexesOperation(index.CreateIndexDefinition()));
+            var filterredResults = results.Where(r => r.Index == index.IndexName).ToArray();
+            return filterredResults.Length == 1 ? filterredResults[0] : null;
```

Comment:
this method should wait for that to be applied in the cluster internally, not externally. It does not make sense to have this method with current behavior.

Reply:
Done